cli/src/main/java/org/jboss/as/cli/Util.java

```diff
@@ -346,6 +346,14 @@ public static boolean isWindows() {
         return WildFlySecurityManager.getPropertyPrivileged("os.name", null).toLowerCase(Locale.ENGLISH).indexOf("windows") >= 0;
     }
 
+    public static boolean isSolaris() {
+        String osName = System.getProperty("os.name");
```

Comment:
We should use the `WildFlySecurityManager.getPropertyPrivileged` as it looks like that's what `isWindows()` and others do.